### PR TITLE
HBASE-18439 Subclasses of o.a.h.h.chaos.actions.Action all use the same logger

### PR DESCRIPTION
**Pull request by ravowlga123**
In org.apache.hadoop.hbase.chaos.actions.* were all using same logger inherited from Action class. So, I have added logger for each class which inherits Action class and also made logger of Action from protected to private. Also, fixed some checkstyle issues of sub-module Action.

### CODE REVIEW

File: hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/BatchRestartRsAction.java [hunk 1]
```diff
@@ -21,14 +21,19 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Restarts a ratio of the running regionservers at the same time
  */
 public class BatchRestartRsAction extends RestartActionBaseAction {
   float ratio; //ratio of regionservers to restart
+  private static final Logger LOG =
+          LoggerFactory.getLogger(BatchRestartRsAction.class);
```
Comment (HorizonNet) [inline hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/BatchRestartRsAction.java]: Should be intended at level 6.

Comment (ravowlga123) [reply hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/BatchRestartRsAction.java]: @HorizonNet I have made the necessary changes. Hopefully, changes made will address the indentation issue.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   2m 42s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 35 new or modified test files.  |
||| _ master Compile Tests _ |
| :green_heart: |  mvninstall  |   8m 34s |  master passed  |
| :green_heart: |  compile  |   0m 35s |  master passed  |
| :green_heart: |  checkstyle  |   0m 21s |  master passed  |
| :green_heart: |  shadedjars  |   6m 18s |  branch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  javadoc  |   0m 19s |  master passed  |
| :blue_heart: |  spotbugs  |   6m 40s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| :green_heart: |  findbugs  |   0m  0s |  master passed  |
||| _ Patch Compile Tests _ |
| :green_heart: |  mvninstall  |   7m 31s |  the patch passed  |
| :green_heart: |  compile  |   0m 37s |  the patch passed  |
| :green_heart: |  javac  |   0m 37s |  the patch passed  |
| :green_heart: |  checkstyle  |   0m 18s |  hbase-it: The patch generated 0 new + 4 unchanged - 29 fixed = 4 total (was 33)  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| :green_heart: |  shadedjars  |   6m  1s |  patch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  hadoopcheck  |  22m 53s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| :green_heart: |  javadoc  |   0m 15s |  the patch passed  |
| :green_heart: |  findbugs  |   0m  0s |  the patch passed  |
||| _ Other Tests _ |
| :green_heart: |  unit  |   1m  9s |  hbase-it in the patch passed.  |
| :green_heart: |  asflicense  |   0m 14s |  The patch does not generate ASF License warnings.  |
|  |   |  66m 26s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-794/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/794 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 59339a23b459 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-794/out/precommit/personality/provided.sh |
| git revision | master / f0f7fae400 |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-794/2/testReport/ |
| Max. process+thread count | 396 (vs. ulimit of 10000) |
| modules | C: hbase-it U: hbase-it |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-794/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/BatchRestartRsAction.java (hunk 1 above)
Comment (HorizonNet) [reply hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/BatchRestartRsAction.java]: Still not level 6. 6 is at the `v` of `private`.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 35s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 35 new or modified test files.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   6m 34s |  master passed  |
| +1 :green_heart: |  compile  |   0m 35s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 20s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m  9s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 14s |  master passed  |
| +0 :ok: |  spotbugs  |   5m 25s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m  0s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 33s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 16s |  hbase-it: The patch generated 0 new + 4 unchanged - 29 fixed = 4 total (was 33)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   5m 11s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  18m 45s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   0m 14s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m  0s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m  3s |  hbase-it in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 13s |  The patch does not generate ASF License warnings.  |
|  |   |  53m 32s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-794/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/794 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 241d42bbc2b1 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-794/out/precommit/personality/provided.sh |
| git revision | master / 2c1d500828 |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-794/3/testReport/ |
| Max. process+thread count | 396 (vs. ulimit of 10000) |
| modules | C: hbase-it U: hbase-it |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-794/3/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**